### PR TITLE
Storyteller - tracking page update

### PR DESCRIPTION
**Pull request by webdevelopland**
What's new:
- design and functionality of tracking page became closer to mocks
- cookies bug fixed

### CODE REVIEW

File: projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts [hunk 1]
```diff
@@ -20,55 +27,66 @@ const MAX_PAUSE_INTERVAL: number = 5 * 60 * 1000; // 5 min
...
-        tracks.sort((a, b) => {
-          return Math.sign(a.getEndedMs() - b.getEndedMs());
-        });
+  }
+
+  storySelected(story: Story): void {
+    this.isTrackingLoading = true;
+    this.trackSub = this.firebaseService.getTracks(story.getId()).subscribe(tracks => {
+      tracks.sort((a, b) => {
+        return Math.sign(a.getEndedMs() - b.getEndedMs());
```
Comment (oferb) [inline projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts]: Not sure you need `Math.sign()`. Isn't it positive/negative?

File: projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts [hunk 2]
```diff
@@ -20,55 +27,66 @@ const MAX_PAUSE_INTERVAL: number = 5 * 60 * 1000; // 5 min
...
-        // Tracks are saved each minute automatically.
-        // To not show 9000 short tracks, combine the several tracks to one.
-        this.tracks = [];
-        let startMs;
-        let storyDurationMs: number = 0;
-        tracks.forEach((track, i) => {
-          const trackDurationMs: number = track.getEndedMs() - track.getStartedMs();
-          storyDurationMs += trackDurationMs;
+      // Tracks are saved each minute automatically.
+      // To not show 9000 short tracks, combine the several tracks to one.
```
Comment (oferb) [inline projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts]: :-))

File: projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts (hunk 1 above)
Comment (webdevelopland) [reply projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts]:
```
Math.sign(*number < 0*) = -1
Math.sign(0) = 0
Math.sign(*number > 0*) = 1
```

File: projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts (hunk 2 above)
Comment (webdevelopland) [reply projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts]: :-))

**Pull request merged**